tests/models/rembert/test_tokenization_rembert.py

```diff
@@ -0,0 +1,545 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the RemBert tokenizer. """
+
+
+import tempfile
+import unittest
+
+from tests.test_tokenization_common import AddedToken, TokenizerTesterMixin
+from transformers import RemBertTokenizer, RemBertTokenizerFast
+from transformers.testing_utils import get_tests_dir, require_sentencepiece, require_tokenizers
+
+
+SENTENCEPIECE_UNDERLINE = "▁"
+SPIECE_UNDERLINE = SENTENCEPIECE_UNDERLINE  # Kept for backward compatibility
+
+SAMPLE_VOCAB = get_tests_dir("fixtures/test_sentencepiece.model")
+
+
+@require_sentencepiece
+@require_tokenizers
+class RemBertTokenizationTest(TokenizerTesterMixin, unittest.TestCase):
+    tokenizer_class = RemBertTokenizer
```

Comment:
anything that is copied from somwhere else needs the `# Copied from` mentions see [here](https://huggingface.co/docs/transformers/v4.35.2/en/pr_checks#check-copies) for doc on that! 🤗